smpe/ZOSMF/vtls/ZWE7APLY.vtl

```diff
@@ -55,7 +55,7 @@
 //*
 //********************************************************************
 //*
-//APPLY    EXEC PGM=GIMSMP,REGION=0M
+//APPLY    EXEC PGM=GIMSMP,REGION=0M,COND=(4,LT)
```

Comment:
as FYI, the point of the pointless condition is simplifying merging all SMPE jobs into a single one and halt on error.

Reply:
I figured it was like that, however, the condition is "pointless" in the workflow :)